IdentityCore/src/broker_operation/request/interactive_token_request/MSIDBrokerOperationInteractiveTokenRequest.m

```diff
@@ -47,7 +49,14 @@ - (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(NSError **)er
     
     if (self)
     {
-        // TODO: implement
+        NSError *localError;
+        _accountIdentifier = [[MSIDAccountIdentifier alloc] initWithJSONDictionary:json error:&localError];
```

Comment:
I think there's a key missing, otherwise you're passing whole json to the MSIDAccountIdentifier

Reply:
It is a flat dictionary (so we can serialize it to query param), all properties are mixed there. MSIDAccountIdentifier will need to pick correct properties from the whole json dict.

Reply:
The problem with this approach is that serialization and deserialization become non-symmetrical. For example, I initialize accountIdentifier with 5 properties, and I want to convert it back to JSON, but I get only 2 back. Additionally, some JSON serializable classes store entire JSON to support additional fields (at least in cache). In that case, account identifier would be serialized to 5 unrelated properties...

Reply:
Decided to keep this logic. Added a comment in code.